src/Symfony/Component/Messenger/DecoratedMessageBus.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Messenger;
+
+/**
+ * A decorated message bus.
+ *
+ * Use this abstract class to created your message bus decorator to specialise your
+ * bus instances and type-hint them.
```

Comment:
I suggest `type-hint against them`, dependents type-hint one of their dependencies against the class